ipamd/ipamd.go

```diff
@@ -590,13 +590,8 @@ func (c *IPAMContext) increaseIPPool() {
 	}
 
 	if c.dataStore.GetENIs() < maxENIs {
-		// c.maxENI represent the discovered maximum number of ENIs
-		if (c.maxENI > 0) && (c.maxENI == c.dataStore.GetENIs()) {
```

Comment:
Do we want to remove `maxENI` from `IPAMContext` as well, since it was only ever used here?

Reply:
Yes, I thought I had.